### PR TITLE
Create CITATION.cff

### PR DESCRIPTION
**Pull request by meghalithic**

### CODE REVIEW

**Review by johnbradley — APPROVED** [review]
LGTM

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    given-names: John
+    orcid: "https://orcid.org/0000-0003-3858-848X"
+  - family-names: Balk
+    given-names: Meghan
+    orcid: "https://orcid.org/0000-0003-2699-3066"
+  - family-names: Lapp
+    given-names: Hilmar
+    orcid: "https://orcid.org/0000-0001-9107-0714"
+identifiers:
+  - description: "This is the BGNN image segmentation workflow managed using snakemake. This workflow contains components to extract image metadata, download images, crop images, and feed images through a segmentation pipeline. The intent is for this to be used as a subworkflow as part of a larger imageomic project."
```
Comment (hlapp) [inline CITATION.cff:17]: This doesn't strike me as a description of (i.e., contextual information for) the identifier (see [guidance doc](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#identifiers)). Instead, perhaps this should go at the top level as [`abstract`](https://github.com/citation-file-format/citation-file-format/blob/main/schema-guide.md#abstract)?

Comment (meghalithic) [reply CITATION.cff:17]: What would you recommend for the description?

Comment (hlapp) [reply CITATION.cff:17]: You mean the description for the identifier? I don't think it needs one (it's the only one), but if you want to put one, I'd just say what it is (snapshot of the repository at a specific version, or at the latest version if you're using the version-independent DOI for the Zenodo record).

**Comment by meghalithic** [comment]
See PR #50

**Pull request merged**